assets/js/phoenix_live_view.js

```diff
@@ -1352,7 +1367,7 @@ class DOMPatch {
         },
         onBeforeNodeDiscarded: (el) => {
           if(el.getAttribute && el.getAttribute(PHX_REMOVE) !== null){ return true }
-          if(el.parentNode !== null && DOM.isPhxUpdate(el.parentNode, phxUpdate, ["append", "prepend"])){ return false }
+          if(el.parentNode !== null && DOM.isPhxUpdate(el.parentNode, phxUpdate, ["append", "prepend"]) && el.id){ return false }
```

Comment:
Theoretically we don't need this change anymore, right?

Reply:
We do, but the reason is subtle.

If our initial render has a bunch of divs and text nodes, and we append with an update that has just one div, we're going to clean the text nodes from the `toEl` but not from the `fromEl`, so they will still remain there.

The number of text nodes will not grow unbounded without this change, but if we want to have the `phx-update` container to only have children with IDs we need this.